CHANGELOG

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.9.1] - 2019-01-16
+### Change
+- You can now hide the visibility levels with the prop `enforceVisibilityLevel`, by default this is set to `true`
```

Comment:
I think you meant the other prop here :)

Reply:
Whoops, thanks for catching that